javascript/selenium-core/scripts/selenium-api.js

```diff
@@ -3135,6 +3174,17 @@ Selenium.prototype.doCaptureEntirePageScreenshot = function(filename, kwargs) {
         height: Math.max(doc.scrollHeight, body.scrollHeight)
     };
 
+    if (kwargs) {
+      var args = parse_kwargs(kwargs);
+      if (args.viewportOnly) {
+        box.width = window.innerWidth;
+        box.height = window.innerHeight;
+      }
+      if (args.background) {
+        doc.style.background = args.background;
```

Comment:
you can't reorder this bit of functionality without also moving the originalBackground variable declaration.